benchmarks/callbacks/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+add_executable(benchmarks_callbacks_SequenceOfRegionTimer)
```

Comment:
It could also be called just `benchmarks_callbacks_RegionTimer` (because you're comparing two versions).

Reply:
Indeed :heavy_check_mark: